src/gc/gc.cpp

```diff
@@ -4291,7 +4318,7 @@ void* virtual_alloc (size_t size)
 
 void virtual_free (void* add, size_t size)
 {
-    GCToOSInterface::VirtualRelease (add, size);
+    GCToOSInterface::VirtualRelease (add);
```

Comment:
Revert this change - the size argument is intentional to allow GC to be used in different environments (in particular on Unix version of CoreRT), even though it is not used in CoreCLR.